pydantic_ai_slim/pydantic_ai/_run_context.py

```diff
@@ -27,10 +27,12 @@ class RunContext(Generic[AgentDepsT]):
     """The model used in this run."""
     usage: Usage
     """LLM usage associated with the run."""
-    prompt: str | Sequence[_messages.UserContent] | None
+    prompt: str | Sequence[_messages.UserContent] | None = None
     """The original user prompt passed to the run."""
     messages: list[_messages.ModelMessage] = field(default_factory=list)
     """Messages exchanged in the conversation so far."""
+    retries: dict[str, int] = field(default_factory=dict)
+    """Number of retries for each tool."""
```

Comment:
Is this number of retries allowed, or number that have happened so far? Whichever it is, can we update the comment to reflect that?

Also, should this use `default_factory=lambda: defaultdict(int)`?